mirrord/operator/src/crd.rs

```diff
@@ -144,3 +145,50 @@ pub struct CopyTargetSpec {
     /// Ignored if [`Target`] is not [`Target::Deployment`].
     pub scale_down: bool,
 }
+
+/// Features and operations that can be blocked by a `MirrordPolicy`.
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
+#[serde(rename_all = "kebab-case")] // StealWithoutFilter -> steal-without-filter in yaml.
+pub enum BlockedFeatures {
+    /// Blocks stealing traffic in any way (without or without filter).
+    Steal,
+    /// Blocks stealing traffic without specifying (any) filter. Client can still specify a
+    /// filter that matches anything.
+    StealWithoutFilter,
+    /// Blocks stealing traffic without an HTTP header filter. (Client can still specify a filter
```

Comment:
imo drop everything besides the first two. it wasn't asked for yet and not sure if that's how we want the options to look like.